docs/guides/qiskit-addons-sqd-get-started.ipynb

```diff
@@ -1877,68 +1877,392 @@
      "text": [
       "\n",
       "Starting configuration recovery iteration 0\n",
-      "Batch 0 subspace dimension: 6400\n",
-      "Batch 1 subspace dimension: 6400\n",
-      "Batch 2 subspace dimension: 6400\n",
-      "Batch 3 subspace dimension: 6400\n",
-      "Batch 4 subspace dimension: 6400\n",
-      "Batch 5 subspace dimension: 6400\n",
-      "Batch 6 subspace dimension: 6400\n",
-      "Batch 7 subspace dimension: 6400\n",
-      "Batch 8 subspace dimension: 6400\n",
-      "Batch 9 subspace dimension: 6400\n",
+      "Batch 0 subspace dimension: 6400\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Batch 1 subspace dimension: 6400\n"
+     ]
+    },
```

Comment:
To do: Work out why some systems render streaming outputs differently. Might be a nice squeaky pass to consolidate them.